ios-nocodesign.cmake

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2014-2016, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_NOCODESIGN_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_NOCODESIGN_CMAKE_ 1)
+endif()
+
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_module_path.cmake")
+include(polly_clear_environment_variables)
+include(polly_init)
+include("${CMAKE_CURRENT_LIST_DIR}/os/iphone-default-sdk.cmake") # -> IOS_SDK_VERSION
+
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+  "iOS ${IOS_SDK_VERSION} Universal (iphoneos + iphonesimulator) / \
+${POLLY_XCODE_COMPILER} / \
+c++14 support"
+  "Xcode"
+)
+
+include(polly_common)
+include(polly_fatal_error)
+
+# Fix try_compile
+include(polly_ios_bundle_identifier)
+set(CMAKE_MACOSX_BUNDLE YES)
+
+# Verify XCODE_XCCONFIG_FILE
+set(
+  _polly_xcode_xcconfig_file_path
+    "${CMAKE_CURRENT_LIST_DIR}/scripts/NoCodeSign.xcconfig"
+)
+if(NOT EXISTS "$ENV{XCODE_XCCONFIG_FILE}")
+  polly_fatal_error(
+    "Path specified by XCODE_XCCONFIG_FILE environment variable not found"
+    "($ENV{XCODE_XCCONFIG_FILE})"
+    "Use this command to set: "
+    "    export XCODE_XCCONFIG_FILE=${_polly_xcode_xcconfig_file_path}"
+  )
+else()
+  string(
+    COMPARE
+    NOTEQUAL
+    "$ENV{XCODE_XCCONFIG_FILE}"
+    "${_polly_xcode_xcconfig_file_path}"
+    _polly_wrong_xcconfig_path
+  )
+  if(_polly_wrong_xcconfig_path)
+    polly_fatal_error(
+        "Unexpected XCODE_XCCONFIG_FILE value: "
+        "    $ENV{XCODE_XCCONFIG_FILE}"
+        "expected: "
+        "    ${_polly_xcode_xcconfig_file_path}"
+    )
+  endif()
+endif()
+
+# 32 bits support was dropped from iPhoneSdk11.0
+if(IOS_SDK_VERSION VERSION_LESS "11.0")
+  set(IPHONEOS_ARCHS armv7;armv7s;arm64)
+  set(IPHONESIMULATOR_ARCHS i386;x86_64)
+else()
+  polly_status_debug("iPhone11.0+ SDK detected, forcing 64 bits builds.")
+  set(IPHONEOS_ARCHS arm64)
+  set(IPHONESIMULATOR_ARCHS x86_64)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/compiler/xcode.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/os/iphone.cmake")
+if(IOS_SDK_VERSION VERSION_LESS "11.0")
+  include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx11.cmake")
+else()
+  include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx14.cmake")
+endif()
```

Comment:
Actually I would like your opinion here.
In my project, the C++ dialect is imposed inside the main CMakeLists.txt (so that cxx11.cmake and cxx14.cmake are just added on top of existing flags).
Do you think that settings c++11 or c++14 flags belongs to the iOS toolchain ? 

Note : c++14 was possible also with iOS 10 (I do not know when it was first supported).

Reply:
> Do you think that setting c++11 or c++14 flags belongs to the iOS toolchain ?

Yes, I think C++ standard belongs to toolchain. In general it should be global to avoid potential ODR violations:
* http://cgold.readthedocs.io/en/latest/tutorials/libraries/symbols/odr-global.html

Reply:
ok